### DISCUSSION TITLE
Request to Add Support for M3U8 Links in Radio Functionality

### DISCUSSION THREAD

**Discussion by limitcool** [navidrome/navidrome#2952] [Ideas]
I've noticed that the radio functionality of this open-source project currently supports MP3 links exclusively. To enhance the user experience and expand content compatibility, I propose the addition of support for M3U8 streaming links. This feature would enable users to connect to a wider range of radio stations and audio content, thereby increasing the project's versatility and utility.

Could the maintainers or development team consider evaluating the feasibility of this addition and provide feedback on this request?

**Comment by andrewdunndev** [comment]
I think this may have already been accomplished via #2187 and #2108 by @kgarner7. I'd love these features as well!

**Comment by mattn** [comment]
I'm using navidrome with my small app. You can register URLs with this proxy.

https://github.com/mattn/hls2mp3
